src/SDKs/StorageSync/Management.StorageSync/Microsoft.Azure.Management.StorageSync.csproj

```diff
@@ -6,15 +6,15 @@
     <PropertyGroup>
         <PackageId>Microsoft.Azure.Management.StorageSync</PackageId>
         <Description>Provides management capabilities for Azure Storage Sync.</Description>
-        <Version>2.8.0-preview</Version>
+        <Version>2.9.0</Version>
```

Comment:
><Version>2.9.0</Version> [](start = 8, length = 24)

@SDK team , Please help confirm the version. we are trying to provide a stable version number.

Reply:
Stable version here should be `3.0.0`
Please update the AssemblyInfo.cs file too